src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/AdjudicationIncidentParty.kt

```diff
@@ -37,6 +40,41 @@ class AdjudicationIncidentPartyId(
 
 @Entity
 @Table(name = "AGENCY_INCIDENT_PARTIES")
+@NamedEntityGraph(
+  name = "full-adjudication",
+  attributeNodes = [
+    NamedAttributeNode(value = "actionDecision"),
+    NamedAttributeNode(value = "incident", subgraph = "full-incident"),
+    NamedAttributeNode(value = "offenderBooking", subgraph = "booking-offender"),
+  ],
+  subgraphs = [
+    NamedSubgraph(
+      name = "booking-offender",
+      attributeNodes = [
+        NamedAttributeNode("offender", subgraph = "offender-with-gender"),
+        NamedAttributeNode("location"),
+        NamedAttributeNode("createLocation"),
```

Comment:
needs investigation, no idea why it is eager loading this when they are never referenced - so this at least adds them to the JOIN